src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -189,22 +307,86 @@ trait GraphDAO {
 
     val parameters = ctor.asMethod.paramLists.head.map { paramSymbol =>
       val paramName = paramSymbol.name.decodedName.toString.trim
-      val prop = vertexProperties.get("_" + paramName).orElse(vertexProperties.get(paramName)) map { value =>
-        if (paramSymbol.typeSignature =:= typeOf[DateTime]) value match {
-          case date: Date => new DateTime(date)
-          case _ => throw new RawlsException(s"org.joda.time.DateTime property [${paramName}] of class [${classT.fullName}] does not have a java.util.Date value in ${vertexProperties}")
+      val vertexProperty: Option[Any] = Option(vertex.getProperty(paramName))
+      paramSymbol.typeSignature match {
+
+        case dateSymbol if dateSymbol =:= typeOf[DateTime] =>
```

Comment:
http://stackoverflow.com/questions/3427345/what-do-and-mean-in-scala-2-8-and-where-are-they-documented